torch/nn/modules/activation.py

```diff
@@ -1304,10 +1304,11 @@ def __init__(self, dim: Optional[int] = None) -> None:
         self.dim = dim
 
     def __setstate__(self, state):
-        self.__dict__.update(state)
         if not hasattr(self, 'dim'):
             self.dim = None
 
+        super().__setstate__(state)
```

Comment:
nit: Why put this at the end? Probably doesn't matter, but I expected it at the beginning :p

Reply:
Sure, doesn't seem like it matters but moved it to the beginning